### DISCUSSION TITLE
blog/%ED%9A%8C%EA%B3%A0-%EC%A0%80%ED%9D%AC-%EB%B8%94%EB%A1%9C%EA%B7%B8-%EC%A0%95%EC%83%81-%EC%98%81%EC%97%85-%ED%95%A9%EB%8B%88%EB%8B%A4-10b0af8d-0b40-4275-a115-b72034065089

### DISCUSSION THREAD

**Discussion by giscus** [starcradle101/hoonie-log#43] [Comments]
# blog/%ED%9A%8C%EA%B3%A0-%EC%A0%80%ED%9D%AC-%EB%B8%94%EB%A1%9C%EA%B7%B8-%EC%A0%95%EC%83%81-%EC%98%81%EC%97%85-%ED%95%A9%EB%8B%88%EB%8B%A4-10b0af8d-0b40-4275-a115-b72034065089

프랑켄슈타인은 전기를 머금고 햇빛 아래로 걸어나왔다.

https://hoonie-log.vercel.app/blog/%ED%9A%8C%EA%B3%A0-%EC%A0%80%ED%9D%AC-%EB%B8%94%EB%A1%9C%EA%B7%B8-%EC%A0%95%EC%83%81-%EC%98%81%EC%97%85-%ED%95%A9%EB%8B%88%EB%8B%A4-10b0af8d-0b40-4275-a115-b72034065089

**Comment by junny97** [comment]
깔끔허이
